source/app/index.js

```diff
@@ -182,29 +198,65 @@ function saveCurrent() {
 function setMenu() {
     const menuTemplate = [{
         label: "File",
-        submenu: [{
+        submenu: [
+            {
                 label: "New",
-                click: () => createWindow()
+                click: () => createWindow(),
+                accelerator: "CmdOrCtrl+N"
             },
             {
                 label: "Open",
-                click: openInCurrent
+                click: openInCurrent,
+                accelerator: "CmdOrCtrl+O"
             },
             {
                 label: "Save",
-                click: saveCurrent
+                click: saveCurrent,
+                accelerator: "CmdOrCtrl+S"
             }
         ]
+    }, {
+        label: 'Edit',
+        submenu: [
+            { role: 'undo' },
+            { role: 'redo' },
+            { type: 'separator' },
+            { role: 'cut' },
+            { role: 'copy' },
+            { role: 'paste' },
+            { role: 'pasteandmatchstyle' },
+            { role: 'delete' },
+            { role: 'selectall' }
+        ]
+    }, {
+        label: 'View',
+        submenu: [
+            { role: 'reload' },
+            { role: 'forcereload' },
+            { role: 'toggledevtools' },
+            { type: 'separator' },
+            { role: 'togglefullscreen' }
+        ]
+    }, {
+        role: 'window',
+        submenu: [
+            {role: 'minimize'},
+            {role: 'close'}
+        ]
     }];
     if (process.platform === "darwin") {
         menuTemplate.unshift({
             label: app.getName(),
-            submenu: [{
-                    role: "about"
-                },
-                {
-                    role: "quit"
-                }
+            submenu: [
+                { role: 'about' },
+                { type: 'separator' },
+                { role: 'services', submenu: [] },
```

Comment:
Services submenu is a standard menu in all macOS apps that provides an integration for 3rdparty apps to implement services based on user input. for example Autmators, Text readers, accessibility helpers etc use this. For example when you select some text from Mead and choose "Create Snippet in Dash":

![image](https://user-images.githubusercontent.com/768052/27254940-0b1cbe6e-539c-11e7-8bce-9727295563ea.png)

The selected text goes to Dash:

![image](https://user-images.githubusercontent.com/768052/27254944-2fba46f6-539c-11e7-96eb-f1a026bef4c0.png)